src/docs/sphinx/quickstart_guide/the_code.rst

```diff
@@ -104,9 +104,11 @@ and implementation files, as well as user documentation, examples and tests.
 
 Axom has the following built-in third party libraries:
```

Comment:
To reduce maintenance burden, should this list be replaced with a link to the similar list in the Dev Guide TPL section?

Reply:
I am not sure what it was bringing to that section. i removed it.